homeTask_10/task2.js

```diff
@@ -0,0 +1,33 @@
+function fetchToDo() {
+  return new Promise((resolve, reject) => {
```

Comment:
remove this Promise, fetch already return Promise

Reply:
done